src/demo-app/ripple/ripple-demo.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  Component,
+  ViewChild,
+} from '@angular/core';
+import {MdButton} from '@angular2-material/button/button';
+import {MdCard} from '@angular2-material/card/card';
+import {MdCheckbox} from '@angular2-material/checkbox/checkbox';
+import {MdIcon} from '@angular2-material/icon/icon';
+import {MdInkRipple} from '@angular2-material/ripple/ripple';
+import {MdInput} from '@angular2-material/input/input';
+import {MdRadioButton, MdRadioGroup} from '@angular2-material/radio/radio';
+import {
+  MdUniqueSelectionDispatcher
+} from '@angular2-material/core/coordination/unique-selection-dispatcher';
+
+@Component({
+  moduleId: module.id,
+  selector: 'ripple-demo',
+  templateUrl: 'ripple-demo.html',
+  styleUrls: ['ripple-demo.css'],
+  providers: [MdUniqueSelectionDispatcher],
+  directives: [
+      MdButton, MdCard, MdCheckbox, MdIcon, MdInput, MdRadioButton, MdRadioGroup, MdInkRipple,
```

Comment:
Can you switch these to the MD_XXX_DIRECTIVES constants? We're trying to push for people to always use those.